core/server/services/settings/validate.js

```diff
@@ -90,7 +90,20 @@ _private.validateData = function validateData(object) {
         };
 
         _.each(object.data, (value, key) => {
-            // CASE: short form e.g. data: tag.recipes
+            if (['resource', 'type', 'limit', 'order', 'include', 'filter', 'status', 'visibility', 'slug', 'redirect'].indexOf(key) !== -1) {
```

Comment:
Hm i can also remove this extra protection here for now.

This avoids:

```
data:
  resource: tags
```

Disadvantage is: you can't use these fields as data longform name.

e.g.

```
    data:
      resource:
        resource: tags
```

Is not allowed.

Reply:
This seems sensible.

Reply:
agree